turbo.json

```diff
@@ -5,30 +5,29 @@
       "outputs": ["coverage/**/*"],
       "dependsOn": ["^build"]
     },
-    "//#lint": {},
+
+    // lint tasks
+    "lint": {},
```

Comment:
We're not using `typescript-eslint` so this works! 👍

(Linting is constrained to single files.)

Reply:
Just to make sure I understand, are you saying we'd need a `dependsOn` if we were using `typescript-eslint`? Or are you referring to the fact that `lint`, rather then `//#lint` is kicking off the lint tasks now, and somehow `typescript-eslint` is relevant to that?

Reply:
Yeah, just a `dependsOn`. If we were using `typescript-eslint` we would have to get builds to run first.